src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java

```diff
@@ -83,6 +82,13 @@ public List<PatientRecordDto> getRecords(
         return result.getContent();
     }
 
+    @PreAuthorize("hasRole('" + SecurityConstants.ROLE_ADMIN + "') or @securityUtils.isMemberOfInstitution(#institutionKey)")
+    @GetMapping(value="availablePhases", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I was looking at conventions that we use in URLs and we use "-"

Thus here it make sense to use available-phases instead of availablePhases. Moreover, based on our previous discussion I suggest to have:

availablePhases --> used-record-phases or used-phases

Reply:
@palagdan this unfortunately requires also frontend refactoring